source/plugin/wgen/modifiers.rst

```diff
@@ -6,7 +6,8 @@ WorldGeneratorModifiers
     org.spongepowered.api.CatalogType
     org.spongepowered.api.GameRegistry
     org.spongepowered.api.data.DataContainer
-    org.spongepowered.api.world.gen.WorldGeneratorModifier
+    org.spongepowered.api.event.game.GameRegistryEvent
+    org.spongepowered.api.event.game.WorldGeneratorModifier
```

Comment:
Wrong import: Should be `org.spongepowered.api.world.gen.WorldGeneratorModifier`